### DISCUSSION TITLE
prism support inheritance in model ?

### DISCUSSION THREAD

**Discussion by cgarrotti** [prisma/prisma#20505] [Q&A] [answered]
### Question

prism support inheritance in model ?

### How to reproduce (optional)

prism support inheritance in model ?

### Information about Prisma Schema, Client Queries and Environment (optional)

```js
// Add your schema.prisma and relevant query code
```

- OS: 
- Database: 
- Node.js version: 

- Run `prisma -v` to see your Prisma version and paste it

**Comment by ludralph** [comment, marked as answer]
Hi @cgarrotti 👋 

At the moment, Prisma does not have support for model inheritance. However, we do have a feature request for it.

- https://github.com/prisma/prisma/issues/12604

You can add your use case to the feature request and leave a 👍  to help our engineers prioritize it.

If this answers your question, it would be great if you could **mark this Discussion as answered** to indicate that it has been resolved. 

Otherwise please let us know how else we can help you further or close the Discussion if it was resolved in some other way 🙏

**Reply by ludralph** [reply]
Hi @cgarrotti 

Do you have any more questions around this issue? If no, it would be really helpful if you could quickly **mark this Discussion as _answered_** to indicate that your issue has been resolved. 

In case you do have any outstanding questions, feel free to let us know and we'd be happy to help!
